app/commands/git/sync_concept_exercise.rb

```diff
@@ -47,8 +47,6 @@ def update_contributors!
     end
 
     def exercise_needs_updating?
-      return false if synced_to_head?
```

Comment:
This was not needed as the synced_to_head check was already done at the level of the track syncing (which calls this file's class)